certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificate/persistence/repository/MessageEntitySpecification.java

```diff
@@ -0,0 +1,34 @@
+package se.inera.intyg.certificateservice.infrastructure.certificate.persistence.repository;
+
+import java.time.LocalDateTime;
+import org.springframework.data.jpa.domain.Specification;
+import se.inera.intyg.certificateservice.domain.message.model.Author;
+import se.inera.intyg.certificateservice.domain.message.model.Forwarded;
+import se.inera.intyg.certificateservice.infrastructure.certificate.persistence.entity.MessageEntity;
+
+public class MessageEntitySpecification {
+
+  private MessageEntitySpecification() {
+  }
+
+  public static Specification<MessageEntity> sentEqualsAndGreaterThan(LocalDateTime from) {
+    return (root, query, criteriaBuilder) ->
+        criteriaBuilder.greaterThanOrEqualTo(root.get("sent"), from.toLocalDate().atStartOfDay());
```

Comment:
Maybe this should be moved to WC so filters already are sent start of day? It's necessary to filter correctly when from and to is set to same day as sent date

Reply:
I think its fine to handle it in CS, since we are not manipulating the value in other ways than reading the startOfDay wich could be consider as "bussiness logic" that CS should handle.